SQL/database_changelog.txt

```diff
@@ -1,17 +1,32 @@
 Any time you make a change to the schema files, remember to increment the database schema version. Generally increment the minor number, major should be reserved for significant changes to the schema. Both values go up to 255.
 
-The latest database version is 7.0; The query to update the schema revision table is:
+The latest database version is 7.1; The query to update the schema revision table is:
 
-INSERT INTO `schema_revision` (`major`, `minor`) VALUES (7, 0);
+INSERT INTO `schema_revision` (`major`, `minor`) VALUES (7, 1);
```

Comment:
these values are also defined in code somewhere, so need to be updated there

Reply:
updated